lib/pf/mfa/Akamai.pm

```diff
@@ -476,6 +476,7 @@ sub redirect_info {
     };
     # Set in the cache that the user did the redirection on Akamai MFA
     cache->set($username."mfapreauth", $TRUE, normalize_time($self->cache_duration) * 2);
+    cache->set($username."mfapostauth", $FALSE, normalize_time($self->cache_duration) * 2);
```

Comment:
I'm going to be mr. picky but this is copy/pasted between the 2 MFA modules, please centralize the logic of success/failure MFA calls in the cache to the base MFA module